docs/rst/getting_started/tutorial.rst

```diff
@@ -6,55 +6,55 @@
 Example of usage
 ################
 
-This example will show how to monitoring a DDS network using Fast DDS Monitor and how to understand the different
+This example will show how to monitoring a DDS network using *Fast DDS Monitor* and how to understand the different
 application features and configurations.
 
 *******************************
 Fast DDS with Statistics module
 *******************************
 
-In order to show the Fast DDS Monitor running and monitoring a real DDS network, we are going to use a Fast DDS
+In order to show the *Fast DDS Monitor* running and monitoring a real DDS network, this tutorial uses a *Fast DDS*
 example to create a simple and understandable DDS network.
-The example proposed by this tutorial is using :code:`DDSHelloWorldExample` of Fast DDS repository.
-Be aware that the statistics module is not compiled and used by default by Fast DDS but it has to be specifically
-configured to run and send statistical data of an specific entity.
+The example proposed by this tutorial is using :code:`DDSHelloWorldExample` of *Fast DDS* repository.
+Be aware that the statistics module is not compiled and used by default by *Fast DDS* but it has to be specifically
+configured to run to send statistical data of an specific entity.
 
 In order to execute this minimum DDS scenario where each entity publish its statistical data, follow these steps:
 
-#. Compile Fast DDS library with CMake option :code:`FASTDDS_STATISTICS` to activate the statistics module.
-#. Compile Fast DDS library with CMake option :code:`COMPILE_EXAMPLES` to build the examples.
-#. Has an scenario with Fast DDS, Fast DDS Statistics Backend and Fast DDS Monitor built.
-#. Use Environmental Variable :code:`FASTDDS_STATISTICS` to activate the statistics writers in the DDS execution.
+#. Compile *Fast DDS* library with CMake option :code:`FASTDDS_STATISTICS` to activate the statistics module.
+#. Compile *Fast DDS* library with CMake option :code:`COMPILE_EXAMPLES` to build the examples.
+#. Have an working environment with *Fast DDS*, *Fast DDS Statistics Backend* and *Fast DDS Monitor* built.
```

Comment:
#. Have a working environment with *Fast DDS*, *Fast DDS Statistics Backend* and *Fast DDS Monitor* built.